src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough7.java

```diff
@@ -0,0 +1,117 @@
+/*
+FallThrough
+checkLastCaseGroup = true
+reliefPattern = (default)falls?[ -]?thr(u|ough)
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;
+
+public class InputFallThrough7 {
+
+        public void method4() {
+        int c;
+        switch (5) {
+            case 5:
+            case 4:
+            case 3:
+                c = 4;
+            // fall through
+            case 2: // violation 'Fall through from the last branch of the switch statement'
+            case 1:
+            default:
+                c = 9;
+        }
+    }
+
+    public void method() {
+        int c;
+        switch (5) {
+            case 5:
+            case 4:
+            case 3:
+                c = 4;
+                break;
+            case 2: // violation 'Fall through from the last branch of the switch statement'
+            case 1:
+            default:
+                c = 9;
+        }
+    }
+
+    public void method2() {
+        int c;
+        switch (5) {
+            case 5:
+            case 4:
+            case 3:
+                c = 4;
+            case 2: // 2 violations
```

Comment:
@romani Another reason to move the violation line. #13088

Reply:
yes, lets make update in separate issue/PR.